core/src/services/gcs/pager.rs

```diff
@@ -97,6 +114,16 @@ impl oio::Page for GcsPager {
                 continue;
             }
 
+            // exclude the inclusive start_after itself
+            if self
```

Comment:
We can use something like the following: 

```rust
let path = build_rel_path(&self.core.root, &object.name);
if self.start_after == Some(path) {
    continue
}

...

let de = oio::Entry::new(&path, meta);
```

Reply:
Thanks, I just forgot relative to root.